credit-details-service/docker/native/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ghcr.io/graalvm/graalvm-ce:java11-21.0.0 AS GRAALVM
```

Comment:
Creating custom Dockerfiles defeats one feature of the Gradle plugin: Having the Dockerfiles generated (and thus kept maintained). Why did you decide to go custom?

Reply:
Add comment to make clear that the Dockerfile is only used for Docker Compose purposes